v2/components/schemas/Publisher.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: Information about the publisher of the snap.
+properties:
+  display-name: 
+    type: string
+    description: |-
+      The user name associated with the account.
+  id:
+    type: string
+    description: |-
+      The developer ID associated with the account.
+  username:
+    type: string
+  validation:
+    type: string
+    enum:
+      - verified
+      - unproven
```

Comment:
There's also `starred`, for instance the `terraform` snap has this. See https://forum.snapcraft.io/t/star-developers/30714